src/site/stages/build/process-cms-exports/schemas/input/media-video.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable camelcase */
+
+const entityReferenceSchema = require('../common/entity-reference');
+
+module.exports = {
+  type: 'object',
+  properties: {
+    name: { $ref: 'GenericNestedString' },
+    thumbnail: {
+      type: 'array',
+      maxItems: 1,
+      items: {
+        type: 'object',
+        properties: {
```

Comment:
You could probably use `usePartialSchema()` here

Reply:
Since I'm using all the properties from the `EntityReference` schema, is there still a need for `usePartialSchema` since the function seems to be used for getting part of a schema's properties?